must-call-checker/src/main/java/org/checkerframework/checker/mustcall/Stream.astub

```diff
@@ -229,3 +293,31 @@ class Scanner {
     @MustCallChoice Scanner(@MustCallChoice ReadableByteChannel source, String charsetName);
     @MustCall({}) Scanner(String source);
 }
+
+class Formatter implements Closeable, Flushable {
+    Formatter();
+    @PolyMustCall Formatter(@PolyMustCall Appendable arg0);
```

Comment:
Why `@PolyMustCall` rather than `@MustCallChoice`? Can you add a comment about it?